mmv1/third_party/terraform/website/docs/guides/provider_reference.html.markdown

```diff
@@ -268,6 +268,38 @@ resource "google_compute_address" "my_address" {
 }
 ```
 
+---
+
+* `skip_terraform_attribution_label` (Optional) Whether to add a label to
+resources indicating that the resource was provisioned using Terraform. When
+set to `true` the label `goog-terraform-provisioned = true` will be added
+automatically to resources, and will be returned in the `terraform_labels`
+and `effective_labels` fields. This makes it possible to distinguish Terraform
+resources when using other tools like Cloud Console or gcloud.
+
+The default value is `true`. No label will be added unless the provider is
+explicitly configured to do so by setting the value to `false`.
```

Comment:
```suggestion
The default value is `true`, and no label will be added. To add the `goog-terraform-provisioned` label,
the provider must be explicitly configured to do so by setting the value to `false`.
```

I found this a little hard to understand that the default was to add nothing- here's a suggestion, not sure it ended up much better though.

I would consider changing this to default to `false` rather than `true`, such as `add_terraform_attribution_label`, as those are generally easier to reason about. Non-blocking though.

Reply:
Yeah, the current name is awkward and that's largely due to the default being `true`; if the default ever becomes `false` it'll be more wieldy.

Hmm, thinking about it I think I'll make the change to `add_terraform_attribution_label` with a default of `false` as part of this PR. It'll only get harder to change in the future.

Reply:
Done.